src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -6,6 +6,7 @@ import {colors} from '../../../utils/colors';
 import {Button} from '../../../components/Button';
 import {BottomSheet} from '../../../components/BottomSheet';
 import AutoFocusTextArea from '../../../components/TextArea/AutoFocusTextArea';
+import dimen from '../../../utils/dimen';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `dimen` is new in this hunk. Ensure that the `dimen` module exists and is correctly exported from the specified path. If it doesn't exist, it will cause a runtime error.
```diff
+ import dimen from '../../../utils/dimen';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
the path is exis